src/custom-operations/parse-schema.ts

```diff
@@ -22,6 +22,14 @@ const customSchemasPathsV2 = [
   '$.components.messages.*',
 ];
 
+const customSchemasPathsV3 = [
+  // operations
+  '$.channels.*.messages.*.payload',
+  '$.components.channels.*.messages.*.payload',
+  '$.components.messages.*.payload',
+  '$.components.schemas.*',
+];
```

Comment:
Just wondering if we should not include headers and examples.

```suggestion
const customSchemasPathsV3 = [
  // channels
  '$.channels.*.messages.*.payload',
  '$.channels.*.messages.*.headers',
  '$.components.channels.*.messages.*.payload',
  '$.components.channels.*.messages.*.headers',
  // operations
  '$.operations.*.messages.*.payload',
  '$.operations.*.messages.*.headers',
  '$.components.operations.*.messages.*.payload',
  '$.components.operations.*.messages.*.headers',
  // messages
  '$.components.messages.*.payload',
  '$.components.messages.*.headers.*',
  '$.components.messages.*.examples.*.headers',
  '$.components.messages.*.examples.*.payload',
  // schemas 
  '$.components.schemas.*',
];
```

Reply:
But does any of these need custom schema parsing? Headers, you right 👍 Examples, those are actual values not schemas 🤔 

Also `'$.operations.*.messages.*.payload',` is not even a thing right? Unless references are resolved the schema parsing?

Reply:
Resolution happens **after** applying custom operations (inclulding schema parsing), right?

Reply:
It did not 😄 

Adapted the paths and tests accordingly.